src/azure-cli/azure/cli/command_modules/cognitiveservices/tests/latest/test_cognitiveservices_command.py

```diff
@@ -48,7 +47,7 @@ def test_cognitiveservices_crud(self, resource_group):
                                    self.check('length(key2)', 32)]).get_output_in_json()
 
         # test to regenerate the keys of a cognitive services account
-        newkeys = self.cmd('az cognitiveservices account keys regenerate -n {sname} -g {rg} --key-name key1').get_output_in_json()  # pylint: disable=line-too-long
+        newkeys = self.cmd('az cognitiveservices account keys regenerate -n {sname} -g {rg} --key-name Key1').get_output_in_json()  # pylint: disable=line-too-long
```

Comment:
For old users, will `key1` works? Could you have a try?

Reply:
I tried, it doesn't. Now I changed to use `get_enum_type(KeyName)` and both `key1` and `Key1` will work.